module/signature/signer_indices_test.go

```diff
@@ -342,17 +344,27 @@ func Test_DecodeSignerIndicesToIdentities(t *testing.T) {
 
 		// create committee
 		identities := unittest.IdentityListFixture(committeeSize, unittest.WithRole(flow.RoleConsensus)).Sort(order.Canonical)
-		signers, err := identities.Sample(uint(numSigners))
+		fullSigners, err := identities.Sample(uint(numSigners))
 		require.NoError(t, err)
+		signers := fullSigners.ToSkeleton()
 
 		// encode
 		signerIndices, err := signature.EncodeSignersToIndices(identities.NodeIDs(), signers.NodeIDs())
 		require.NoError(t, err)
 
 		// decode and verify
-		decodedSigners, err := signature.DecodeSignerIndicesToIdentities(identities, signerIndices)
+		decodedSigners, err := signature.DecodeSignerIndicesToIdentities(identities.ToSkeleton(), signerIndices)
 		require.NoError(t, err)
-		require.Equal(t, signers.Sort(order.Canonical), decodedSigners.Sort(order.Canonical))
+
+		slices.SortFunc(signers, func(lhs, rhs *flow.IdentitySkeleton) bool {
+			return order.IdentifierCanonical(lhs.NodeID, rhs.NodeID)
+		})
+
+		slices.SortFunc(decodedSigners, func(lhs, rhs *flow.IdentitySkeleton) bool {
+			return order.IdentifierCanonical(lhs.NodeID, rhs.NodeID)
+		})
```

Comment:
By construction, the decoded signer set should be canonically ordered. I think it would be beneficial to test this here, i.e. remove the output ordering so the test would fail, if `DecodeSignerIndicesToIdentities` violates the ordering convention

Reply:
updated code in https://github.com/onflow/flow-go/pull/4545/commits/bb6353bd3ce5fe7796f032b67f7c68d4113e2385